src/quemb/kbe/lo_k.py

```diff
@@ -140,12 +154,20 @@ def get_iao_k(Co, S12, S1, S2=None, ortho=True):
 
 def get_pao_k(Ciao, S, S12):
     """
-    Args:
-        Ciao: output of :func:`get_iao`
-        S: ao ovlp matrix
-        S12: valence orbitals projected into ao basis
-    Return:
-        Cpao (orthogonalized)
+
+    Parameters
+    ----------
+    Ciao:
+        output of :code:`get_iao`
```

Comment:
@lweisburn to which `get_iao` should this refer?

Reply:
I think this should be `get_iao_k` in `kbe/lo_k`, which gets iao's across k-points, instead of the `get_iao` in `molbe`

Reply:
makes sense and changed, thank you :)